### DISCUSSION TITLE
Obtaining TTL for a single record when using BlobCache.Secure

### DISCUSSION THREAD

**Discussion by cristhian-zarate** [reactiveui/Akavache#795] [Q&A]
Hi!
I'm using the built-in secure BlobCache location in a UWP application to store values as strings and I need to refresh the TTL of single records when I access them, so when I read a record, it gets assigned the same expiration period it had before.

Is there any way I can get the expiration of a record similar to the `GetCreatedAt` method without having to create my own implementation?

For example, when inserting a record using:
```cs
await blobCache.InsertObject<string>(key, value, TimeSpan.FromSeconds(30));
```
I would be able to get the TTL value with something like this:
```cs
var expiration = await blobCache.GetExpiration(key); // {00:00:30}
```

If there isn't then, is there a security or technical reason to not have this value exposed?

**Comment by anaisbetts** [comment]
There is no technical reason not to expose this, it could be added to the existing feature request
